packages/client/src/utils/Scaffold.ts

```diff
@@ -30,7 +30,7 @@ export function Scaffold(
     sequence: Step[] = [],
     _checkFn: (() => Promise<boolean>) | (() => boolean),
     { id = '', onError, onDone, onChange }: ScaffoldOptions = {}
-) {
+): any {
```

Comment:
We should prolly think together what the type should be here.

Reply:
👍